core/templates/components/common-layout-directives/navigation-bars/side-navigation-bar.directive.html

```diff
@@ -151,6 +154,18 @@
         <span translate="I18N_SIDEBAR_OPPIA_FOUNDATION"></span>
       </a>
     </li>
+
+    <li ng-class="{'active': $ctrl.currentUrl === '/learn/math'}">
+      <a ng-init="showSubmenu = false" ng-click="showSubmenu = !showSubmenu" class="protractor-mobile-test-classroom">
```

Comment:
You can probably extract the logic inside ng-click into a function in the .ts file and call that here. Generally that's what we do. Then you won't need the ng-init tag.

Reply:
Done!